src/frontends/pytorch/README.md

```diff
@@ -0,0 +1,143 @@
+```
+
+OpenVINO PyTorch Frontend supports extensions. To add an extension, use
+`ov::frontend::pytorch::Frontend::add_extension()` API.
+The following extension types are supported:
+
+* `ov::frontend::tensorflow::ConversionExtension` or `ov::frontend::ConversionExtension` - add a new Loader into the conversion pipeline.
+* `ov::TelemetryExtension` - enable telemetry for the frontend.
+* `ov::BaseOpExtension` - enable support for a custom operation.
+* `ov::detail::SOExtension` - allow support for `ov::BaseOpExtension` extensions loaded from an external library.
+
+## How to Implement Support for a New PyTorch Operation
+
+PyTorch conversion into the OpenVINO opset operations consists of two stages:
+1. Conversion of PyTorch operations to OpenVINO opset using [translators](./src/op/),
+   which directly transforms a PyTorch operation into a sub-graph of the OpenVINO
+   opset. This is a 1->N conversion.
+2. [Internal Transformations](./src/transforms) that transform a sub-graph of
+   operations into a sub-graph of the OpenVINO opset. This is an N->N conversion.
+
+### Operation Translation
+
+Most PyTorch operations can be converted by a single `translator`. The
+dictionary of `translators` is placed in the [op_table.cpp](./src/op_table.cpp)
+file and each translator is located in the [op](../tensorflow_common/src/op/)
+directory:
+
+https://github.com/openvinotoolkit/openvino/blob/491454103ea2f29b242587c6084c19868a879a82/src/frontends/pytorch/src/op_table.cpp#L222-L227
+
+The main rules for translator implementation:
+1. Support dynamic shapes and ranks, undefined types, including future support of new types, such as strings and complex numbers.
+2. Try to maintain the same algorithmic complexity of the decomposition. Fewer operations are usually better.
+3. Use the latest OpenVINO opset version for the translation.
+4. Use helper routines for operation checks and graph construction from `utils.hpp`.
+5. Call `NodeContext::mark_mode()` for each created node.
+
+#### Inplace and Mutable Operations
+
+Some PyTorch operations modify the input tensor rather than the output. For example,
+`aten::add` writes the result of addition to the output, but `aten::add_` writes the result
+to its first input. To correctly convert such an operation:
+* Ensure that the output tensor produced by the translation has the same type and shape as the initial input.
+* Call `NodeContext::mutate_input()` to change the input tensor with the new value.
+
+#### PtFrameworkNode Primitive
+
+`PtFrameworkNode` is used to represent unconverted operation from the original
+model. You can use `FrontEnd::convert_partially()` instead of `Frontend::convert()`
+to get an `ov::Model` containing unconverted operations.
+
+#### Operations Accepting Strings
+
+At the moment strings are not supported by OpenVINO core. Note that in models
+strings are usually constants, so you can extract them as `std::string` directly
+from Python using `NodeContext::const_input<std::string>()`. 
```

Comment:
```suggestion
At the moment,  OpenVINO core does not support strings. However, since strings in models are usually constants, you can extract them as `std::string` directly from Python using `NodeContext::const_input<std::string>()`. 
```